### PR TITLE
Callout in js

### PR DESCRIPTION
**Pull request by gustavo-salazar**
Replacing all the usages of the foundation class `"callout"` for the new `Callout` component.

### CODE REVIEW

File: src/components/SearchResults/ExactMatch/Gene/index.js [hunk 1]
```diff
@@ -58,9 +60,8 @@ ListOfProteins.propTypes = {
   gene: string,
 }; */
 const ExactGeneMatchWrapper = ({ matches, gene }) => (
-  <div className={f('callout', 'warning', 'margin-bottom-medium')}>
-    <span className={f('icon', 'icon-common')} data-icon="&#xf35a;">
-      {' '}
+  <Callout type="warning" customIcon="icon-arrow-alt-circle-right">
```
Comment (matthiasblum) [inline src/components/SearchResults/ExactMatch/Gene/index.js:63]:
```suggestion
  <Callout type="warning" icon="icon-arrow-alt-circle-right">
```

if accepting [this suggestion](https://github.com/ProteinsWebTeam/interpro7-client/pull/567#discussion_r1471279716).

File: src/components/SearchResults/ExactMatch/index.js [hunk 2]
```diff
@@ -39,13 +33,9 @@ class ExactMatchWrapper extends PureComponent /*:: <EMWProps> */ {
...
     const { to, children } = this.props;
     return (
-      <div className={f('callout', 'warning', 'margin-bottom-medium')}>
-        <span className={f('icon', 'icon-common')} data-icon="&#xf35a;">
-          {' '}
-          Found an exact match:
-        </span>{' '}
-        <Link to={to}>{children}</Link>
-      </div>
+      <Callout type="warning" customIcon="icon-arrow-alt-circle-right">
```
Comment (matthiasblum) [inline src/components/SearchResults/ExactMatch/index.js:36]:
```suggestion
      <Callout type="warning" icon="icon-arrow-alt-circle-right">
```

if accepting [this suggestion](https://github.com/ProteinsWebTeam/interpro7-client/pull/567#discussion_r1471279716).

File: src/components/Toast/ToastDisplay/index.js [hunk 3]
```diff
@@ -77,7 +73,7 @@ class ToastDisplay extends PureComponent /*:: <Props, State> */ {
       <ul
         className={styles['toast-display']}
         onMouseEnter={this._handleMouseEnter}
-        onMouseLeave={this._handleMouseLeave}
+        // onMouseLeave={this._handleMouseLeave}
```
Comment (matthiasblum) [inline src/components/Toast/ToastDisplay/index.js:76]: Why not supporting this callback anymore?

File: src/styles/blocks.css [hunk 4]
```diff
@@ -23,3 +23,13 @@
...
+
+.simple-box {
+  position: relative;
+  margin: 0 0 1rem 0;
+  padding: 1rem;
+  border: 1px solid rgba(10, 10, 10, 0.25);
+  border-radius: 0;
+  background-color: var(--colors-nearly-white);
+  color: #0a0a0a;
+}
```
Comment (matthiasblum) [inline src/styles/blocks.css:35]: Couldn't be done with an `Callout` using `alt` and no type/icon?

**Review by matthiasblum — APPROVED** [review]
Looks good to me. 

Made a few comments, but nothing that prevent merging.

File: src/components/Toast/ToastDisplay/index.js (hunk 3 above)
Comment (gustavo-salazar) [reply src/components/Toast/ToastDisplay/index.js:76]: My mistake, I had commented to test some styling and forgot about it

File: src/styles/blocks.css (hunk 4 above)
Comment (gustavo-salazar) [reply src/styles/blocks.css:35]: Yeah, it's possible, but I think was more of a semantic decision. as the components using this style won't necessarily be sending a message, and just need the style, so technically is not a `Callout`

**Pull request merged**